lily/hairpin.cc

```diff
@@ -271,15 +271,18 @@ Hairpin::print (SCM smob)
 
   Real starth = 0;
   Real endh = 0;
+  Real max_ratio = width / 2
+  Real continued_height = height / 3
+  Real continuing_height = 2 * height / 3
```

Comment:
These three values should *not* be hardcoded but instead retrieved from to-be-created grob properties